async-query/src/main/java/org/opensearch/sql/spark/scheduler/model/OpenSearchRefreshIndexJobRequest.java

```diff
@@ -0,0 +1,108 @@
+/*
+ * Copyright OpenSearch Contributors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+package org.opensearch.sql.spark.scheduler.model;
+
+import java.io.IOException;
+import java.time.Instant;
+import lombok.Builder;
+import org.opensearch.core.xcontent.ToXContent;
+import org.opensearch.core.xcontent.XContentBuilder;
+import org.opensearch.jobscheduler.spi.ScheduledJobParameter;
+import org.opensearch.jobscheduler.spi.schedule.Schedule;
+
+/** Represents a job request to refresh index. */
+@Builder
+public class OpenSearchRefreshIndexJobRequest implements ScheduledJobParameter {
+  // Constant fields for JSON serialization
+  public static final String JOB_NAME_FIELD = "jobName";
+  public static final String JOB_TYPE_FIELD = "jobType";
+  public static final String LAST_UPDATE_TIME_FIELD = "lastUpdateTime";
+  public static final String LAST_UPDATE_TIME_FIELD_READABLE = "last_update_time_field";
+  public static final String SCHEDULE_FIELD = "schedule";
+  public static final String ENABLED_TIME_FIELD = "enabledTime";
+  public static final String ENABLED_TIME_FIELD_READABLE = "enabled_time_field";
+  public static final String LOCK_DURATION_SECONDS = "lockDurationSeconds";
+  public static final String JITTER = "jitter";
+  public static final String ENABLED_FIELD = "enabled";
+
+  // name is doc id
+  private final String jobName;
```

Comment:
what is jobName? could u explain more `name is doc id`?

Reply:
JobName is the required field for ScheduledJobParameter interface. The plan is to reuse it for the doc id.